src/RSCoin/Core/Communication.hs

```diff
@@ -286,6 +289,21 @@ getStatisticsId =
         (L.logDebug . sformat ("Statistics id is " % int)) $
     callBank $ P.call (P.RSCBank P.GetStatisticsId)
 
+addMintetteUsingPermission
+    :: WorkMode m
+    => SecretKey
+    -> String
+    -> Int
+    -> m ()
+addMintetteUsingPermission mintetteSK host port = do
+    L.logDebug $ sformat
+        ("Adding mintette " % build % " listening on port " % int)
+        host
+        port
+    let signed = mkWithSignature mintetteSK (host, port)
```

Comment:
Ok, my bad, I agree there is no reason to do it. Then there is only one simple thing to fix in `rscoin-core`.